tests/acceptance/upcoming-project-cards-renders-test.js

```diff
@@ -33,13 +33,15 @@ module('Acceptance | upcoming project cards renders', function(hooks) {
         this.server.create('assignment', {
           id: 1,
           tab: 'upcoming',
-          dcpPublicstatus: 'Filed',
           dcpLupteammemberrole: 'BP',
           dispositions: [],
           project: this.server.create('project', {
-            milestones: [this.server.create('milestone', 'communityBoardReview', {
-              dcpPlannedstartdate: moment().add(32, 'days'),
-            })],
+            dcpPublicStatusSimp: 'Filed',
```

Comment:
this is camel case PublicStatus—is that right?

Reply:
Good catch!! 🙇 It shouldn't be camel cased! THANK YOU!